en/orm/table-objects.rst

```diff
@@ -140,6 +141,13 @@ Event List
 * ``Model.afterDelete``
 * ``Model.afterDeleteCommit``
 
+initialize
+----------
+
+.. php:method:: initialize(Event $event, ArrayObject $data, ArrayObject $options)
```

Comment:
Table classes don't natively listen for this event. The initialize() hook method is used on tables instead. The signature should not have any additional options either.